ra_aid/agent_utils.py

```
@@ -42,6 +44,8 @@
     WEB_RESEARCH_PROMPT_SECTION_PLANNING,
     HUMAN_PROMPT_SECTION_PLANNING,
     WEB_RESEARCH_PROMPT,
+    EXPERT_PROMPT_SECTION_CHAT,
```

Ruff (0.8.2)

`ra_aid.prompts.EXPERT_PROMPT_SECTION_CHAT` imported but unused

Remove unused import

(F401)